packages/image_picker/image_picker_android/android/gradle.properties

```diff
@@ -3,12 +3,12 @@
 #
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-# Default value: -Xmx1024m -XX:MaxPermSize=256m
-# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+# Default value: -Xmx4G -XX:MaxPermSize=256m
```

Comment:
Overzealous find/replace?

Reply:
Actually I did this by hand, just to not have false "positives" later on :P

Reply:
We should probably just remove the lines entirely, rather than have a comment that incorrectly states what the default is.

Reply:
Yep, let me clean this up, no other file has anything like this.

Reply:
Removed all comments from this file.